admin/javascript/src/components/form-builder/tests/form-builder-test.js

```diff
@@ -1,26 +1,27 @@
+/* global jest, describe, beforeEach */
+
 jest.unmock('silverstripe-component');
 jest.unmock('../');
 
-import { FormBuilderComponent } from '../';
+// import { FormBuilderComponent } from '../';
 
 describe('FormBuilderComponent', () => {
-
   describe('getFormSchema()', () => {
-
-    var formBuilder;
+    /*
```

Comment:
I assume you've commented this out to make it clear that no tests are being run, correct?